content/termine.ics/contents.lr

```diff
@@ -57,7 +57,7 @@ event: Freifunk Installation Haus der Katholischen Kirche Ravensburg besprechen
 ----
 ort: Toolbox Bodensee e.V.
 ----
-url:
+url: 
```

Comment:
Willst du die ganzen Whitespace-Änderungen bei den anderen Terminen noch zurück ändern? Dann sieht man das sich da eigentlich nichts geändert hat.

Reply:
Das wird halt immer wieder kommen da das durch lektor generiert wird...

Aber für dich mach ich das doch gerne

Reply:
Ah, du editierst das in dem Admin von Lektor, das erklärt es. Ich hatte die Seite in meinem normalen Editor bearbeitet, der haut die Whitespaces wieder raus.

Das ist natürlich doof, dass das der Editor von Lektor nicht richtig macht, aber kann man wohl nix machen. In dem Fall ist's das nächste Mal wohl wieder drin. Danke trotzdem! :+1: